gdal/gdal_macros.xml

```diff
@@ -0,0 +1,145 @@
+<macros>
+    
+    <xml name="gdal_requirements">
+        <requirements>
+            <requirement type="package" version="2.1.0">gdal</requirement>
+            <requirement type="package" version="3.5.0">geos</requirement>
+            <requirement type="package" version="1.5.90">libjpeg-turbo</requirement>	    
```

Comment:
Are you sure that libjpeg-turbo is needed? It should be automatically pulled in.

Proposal, get the PR in a shape that is passes all the tests. Then remove this line and we see it if its needed :)

Reply:
I try this localy